bin/process

```diff
@@ -120,18 +115,18 @@ function verify1() {
     for batch_id in $(jq -r '.[].batch_id' "${DATA_CONFIG}"); do
         prefix="batch_id=${batch_id}"
         n_data=$(config_get "${batch_id}" "n_data")
-        if ! gsutil -q stat "${input}/${prefix}/*"; then
+        if ! mc stat "internal/${input}/${prefix}" &> /dev/null; then
```

Comment:
Here, we use the `internal` alias, which points to the GCS gateway with the internal set of `ACCESS_KEY`, `SECRET_KEY` and `ENDPOINT`. This is used instead of `s3://` or `gs://` like we're normally used to seeing in airflow.